dotnet-tests/UniffiCS.Tests/TestBigEndianStream.cs

```diff
@@ -1,15 +1,23 @@
-namespace UniffiCS.tests;
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
+using UniffiCS.Tests.gen;
```

Comment:
Github actions are failing on this line. I think you need to prepend `namespace UniffiCS.Tests.gen` in `run_unit_tests.sh`. But now I'm looking at this test again, and it could just be part of `BindingTests` project, using generated `BigEndianStream` code from any of the other fixture, e.g. `using uniffi.sprites`. There is no reason to have a separate project and separate job just for this.